mobile/src/main/java/org/openhab/habdroid/model/Widget.kt

```diff
@@ -60,7 +60,8 @@ data class Widget(
     val yAxisDecimalPattern: String?,
     val switchSupport: Boolean,
     val height: Int,
-    val visibility: Boolean
+    val visibility: Boolean,
+    val inputHint: String
```

Comment:
I'd parse it as an enum where unexpected values are parsed as `null`.

Reply:
Done